plugins/outputs/opentelemetry/opentelemetry.go

```diff
@@ -59,10 +70,17 @@ func (*OpenTelemetry) SampleConfig() string {
 
 func (o *OpenTelemetry) Connect() error {
 	logger := &otelLogger{o.Log}
-
 	if o.ServiceAddress == "" {
 		o.ServiceAddress = defaultServiceAddress
 	}
+	switch o.EncodingType {
+	case "", "protobuf":
+		o.EncodingType = "application/x-protobuf"
+	case "json":
+		o.EncodingType = "application/json"
+	default:
+		return fmt.Errorf("invalid encoding %q", o.EncodingType)
+	}
```

Comment:
The more I read this the more I think we should do

```suggestion
	switch o.EncodingType {
	case "", "protobuf", "json":
		// Do nothing, those are valid
	default:
		return fmt.Errorf("invalid encoding %q", o.EncodingType)
	}
```

and modify the http client (see other comment)...